### DISCUSSION TITLE
Should `ConcatItemsd` require at least 2 keys?

### DISCUSSION THREAD

**Discussion by function2-llx** [Project-MONAI/MONAI#1880] [General]
Hi everyone I'm using `ConcatItemsd` for concatenating the inputs from multi-modalities images. During development sometimes I adjust the number of modalities of inputs, even to single modality. However I found that `ConcatItemsd` in monai currently does not allow this, with an exception raised [here](https://github.com/Project-MONAI/MONAI/blob/master/monai/transforms/utility/dictionary.py#L678-L679). 

This restriction is inconvenient for me and personally I can't find the point of this restriction because both `numpy.concatenate` and `torch.cat` work fine with an input list of length 1. I'd like to hear your opinions on this :)

**Comment by wyli** [comment]
makes sense, I'm converting this into a feature request, thank you!

https://github.com/Project-MONAI/MONAI/issues/1881

**Reply by function2-llx** [reply]
Thanks so much!
